cartoframes/context.py

```diff
@@ -862,11 +861,17 @@ def _encode_geom(geom):
     """Encode geometries into hex-encoded wkb
     """
     from shapely import wkb
-    return ba.hexlify(wkb.dumps(geom)).decode()
+    if (geom is None):
+        return None
+    else:
+        return ba.hexlify(wkb.dumps(geom)).decode()
```

Comment:
A common Python design form is to rely on whether a variable is null or not to avoid having to do the `is null` check. So things like 
```python
a = 'andy'
if a:
    print('`a` is not None')
else:
    print('`a` is None')
```
Could you switch the order of the if/else and rely on whether `geom` being None-valued or not.